elephant/statistics.py

```diff
@@ -132,12 +135,12 @@ def mean_firing_rate(spiketrain, t_start=None, t_stop=None, axis=None):
 
     if not axis or not found_t_start:
         return np.sum((spiketrain >= t_start) & (spiketrain <= t_stop),
-                      axis=axis) / (t_stop-t_start)
```

Comment:
Please don't put any pep8 changes for other code in the commit.  These aren't default pep8 rules anyway (are you using an old version of pep8?).